tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -95,8 +95,8 @@ Status ModularFileSystem::NewAppendableFile(
 }
 
 Status ModularFileSystem::NewReadOnlyMemoryRegionFromFile(
-    const std::string& fname, std::unique_ptr<ReadOnlyMemoryRegion>*
-                                  result /*, TransactionToken* token */) {
+    const std::string& fname, std::unique_ptr<ReadOnlyMemoryRegion>* result,
+    TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `result`.